SECURITY.md

```diff
@@ -206,6 +206,13 @@ patches, understand exact mitigation steps, etc.
   the fix to user's environments; this can include links to external distributor documentation.
 - The Fix Lead will remove the Fix Team from the private security repo.
 
+The reporter of a vulnerability may request early access to fix patches, prior
```

Comment:
Rephrase for clarity: "The reporter of a vulnerability will be granted early access to fix patches upon request". The current phrasing says they can ask for it, but doesn't say whether their request will be granted or not.